docs/API.md

```diff
@@ -481,6 +481,29 @@ formatted as IPv4-mapped IPv6 addresses). By default (-1) this option is not set
 and the platform default value is used.
 ---
 
+| OptName               | Since | Binding | Type      | Units  | Default  | Range  |
+| --------------------- | ----- | ------- | --------- | ------ | -------- | ------ |
+| `SRTO_KMREFRESHRATE`  | 1.3.2 | pre     | `int32_t` | pkts   | 0x1000000| 0..unlimited |
+
+- **[GET or SET]** - number of packets to be transmitted after which the encryption
```

Comment:
- **[GET or SET]** - number of packets to be transmitted after which the Stream Encrypting Key (SEK)